### PR TITLE
Installing 4.4: add IBM Z and IBM Power, both normal and restricted network versions

### PR DESCRIPTION
**Pull request by rh-max**

### CODE REVIEW

File: modules/installation-about-restricted-network.adoc [hunk 1]
```diff
@@ -14,13 +20,15 @@ not infrastructure that the installation program provisions, so your platform se
 limited.
 // maybe point out that you can follow the bare metal installation rules on supported hardware and link to the matrix
 
+ifndef::ibm-power[]
```
Comment (bmcelvee) [inline modules/installation-about-restricted-network.adoc:23]: Is this content okay to include in Z or should it also be ifdef'd out?

**Comment by bmcelvee** [comment]
One question, and a request to squash commits. :) LGTM. Just confirming this applies to 4.4+, correct? 

@kalexand-rh - FYI - installation content

File: modules/installation-about-restricted-network.adoc (hunk 1 above)
Comment (rh-max) [reply modules/installation-about-restricted-network.adoc:23]: @bmcelvee That's right, this needs to be absent for IBM Power only. It should stay for IBM Z.

**Comment by rh-max** [comment]
@bmcelvee These docs will appear in 4.5+, but for the sake of this PR, consider that they won't. So no cherry-picking to anywhere at this point, just merge to `enterprise-4.4`. However, we've scheduled the merging & publishing with Vikram, so you don't need to do anything on this, Vikram will. Thanks!

**Pull request merged**